demisto_sdk/common/scripts/update_id_set.py

```diff
@@ -1,19 +1,35 @@
+# !/usr/bin/env python
+import itertools
 import os
 import glob
 import json
 from collections import OrderedDict
 from multiprocessing import Pool, cpu_count
+from distutils.version import LooseVersion
 import time
 
 # TODO: do not import *
```

Comment:
hessing what is this ?

Reply:
everything you see here has been copied wholesale from the PR linked in the PR title above so I have no idea